packages/shorebird_cli/lib/src/doctor/validators/shorebird_flutter_validator.dart

```diff
@@ -14,38 +15,56 @@ class ShorebirdFlutterValidator extends DoctorValidator {
 
   @override
   Future<List<ValidationIssue>> validate() async {
+    final issues = <ValidationIssue>[];
+
     if (!ShorebirdPaths.flutterDirectory.existsSync()) {
       final message =
           'No Flutter directory found at ${ShorebirdPaths.flutterDirectory}';
-      return [
+      issues.add(
         ValidationIssue(
           severity: ValidationIssueSeverity.error,
           message: message,
         ),
-      ];
+      );
     }
 
     if (!await _flutterDirectoryIsClean()) {
-      return [
+      issues.add(
         ValidationIssue(
           severity: ValidationIssueSeverity.warning,
           message: '${ShorebirdPaths.flutterDirectory} has local modifications',
         ),
-      ];
+      );
     }
 
     if (!await _flutterDirectoryTracksStable()) {
       final message =
           '${ShorebirdPaths.flutterDirectory} is not on the "stable" branch';
-      return [
+      issues.add(
+        ValidationIssue(
+          severity: ValidationIssueSeverity.warning,
+          message: message,
+        ),
+      );
+    }
+
+    final shorebirdFlutterVersion = await _shorebirdFlutterVersion();
+    final pathFlutterVersion = await _pathFlutterVersion();
+
+    if (shorebirdFlutterVersion != pathFlutterVersion) {
+      final message = 'Shorebird Flutter and the Flutter on your path are '
+          'different versions '
+          '($shorebirdFlutterVersion vs $pathFlutterVersion)';
```

Comment:
```suggestion
      final message = '''
The shorebird Flutter version and host Flutter version are different.
Shorebird Flutter: $shorebirdFlutterVersion
Host Flutter: $pathFlutterVersion''';
```

Minor but thoughts on adjusting this to be a bit more explicit about which is which?

Reply:
Done.